Python/peephole.c

```diff
@@ -306,18 +306,11 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
             case LOAD_CONST:
                 cumlc = lastlc + 1;
                 if (nextop != POP_JUMP_IF_FALSE  ||
-                    !ISBASICBLOCK(blocks, op_start, i + 1)) {
```

Comment:
The changes in this file are to revert the old change (85ed1712e428f93408f56fc684816f9a85b0ebc0).

Reply:
The current code is more reliable (it works if `const` contains some unusual). It is better to keep it.